server/router/activeride.ts

```diff
@@ -8,11 +8,23 @@ const router = express.Router();
 AWS.config.update(config);
 const docClient = new AWS.DynamoDB.DocumentClient();
 
+type ActiveRide = {
+  id: string,
+  startLocation: string,
+  endLocation: string,
+  startTime: number,
+  endTime: number,
+  isScheduled: boolean,
+  riderID: string,
+  driverID: string | null,
+  repeatsOn: string[] | null,
```

Comment:
Is this active ride functioning as a ride by itself, or the ride schedule for a given rider? Because if this repeats, the driver is not guaranteed to be the same every time.

Reply:
both at the moment, i didn't think much about repeating rides yet as i wanted to get the basic ride functionality done first, but yeah ill probably have to revise the schema sometime to accommodate for it